packages/api/pages/api/languages/index.ts

```diff
@@ -1,7 +1,101 @@
-import { NextApiRequest, NextApiResponse } from 'next';
+import * as z from 'zod';
+import { Prisma } from '../../../prisma/client';
+import {
+  GetLanguagesResponseBody,
+  PostLanguageRequestBody,
+  PostLanguageResponseBody,
+} from '@translation/api-types';
+import { ApiRequest, ApiResponse } from '../../../helpers';
+import { client } from '../../../db';
+import { languageSchema } from './schemas';
 
-export default async function (req: NextApiRequest, res: NextApiResponse) {
-  return res.status(200).json({
-    data: null,
-  });
+const postRequestSchema: z.ZodType<PostLanguageRequestBody> = z.object({
+  data: languageSchema(),
+});
+
+export default async function (
+  req: ApiRequest,
+  res: ApiResponse<GetLanguagesResponseBody | PostLanguageResponseBody>
+) {
+  switch (req.method) {
+    case 'GET': {
+      const languages = await client.language.findMany();
```

Comment:
There is a lot in these routes that could be DRY'd up across routes, but I'm going to save that for another PR. The basic flow here is:

1. Routes in nextjs are tied to the file path
2. In the handler, we first branch based on the method, and return 405 for unsupported methods
3. If the request has a body, we use zod to parse and validate it, returning 409 or 422 depending on the error according to JSON:API
4. Then we use Prisma to either fetch or create resources
5. Then return the response with the right code depending on if any errors were thrown.